src/controller/response/balance-response.ts

```diff
@@ -62,6 +62,32 @@ export default interface BalanceResponse extends BaseUserResponse {
   lastTransferId: number;
 }
 
+/**
+ * @typedef {object} UserTypeTotalBalanceResponse
+ * @property {string} userType - The user type
+ * @property {DineroObjectResponse} totalPositive - The total amount of positive balance for this user type
+ * @property {DineroObjectResponse} totalNegative - The total amount of negative balance for this uer type
+ */
+export interface UserTypeTotalBalanceResponse {
+  userType: UserType,
+  totalPositive: DineroObjectResponse,
+  totalNegative: DineroObjectResponse,
+}
+
+/**
+ * @typedef {object} TotalBalanceResponse
+ * @property {string} date - Date at which this total balance was calculated
+ * @property {number} totalPositive - The total amount of positive balance in SudoSOS
+ * @property {number} totalNegative - The total amount of negative balance in SudoSOS
+ * @property {UserTypeTotalBalanceResponse} userTypeBalances - The total balances for the different user types
```

Comment:
all missing a `.required`

```suggestion
 * @property {string} date.required - Date at which this total balance was calculated
 * @property {number} totalPositive.required - The total amount of positive balance in SudoSOS
 * @property {number} totalNegative.required - The total amount of negative balance in SudoSOS
 * @property {UserTypeTotalBalanceResponse} userTypeBalances.required - The total balances for the different user types
```